### PR TITLE
Add nextcloud

### PR DESCRIPTION
**Pull request by domi20**
Add nextcloud to Backup section

### CODE REVIEW

File: _data/backup.yml [hunk 1]
```diff
@@ -136,7 +136,14 @@ websites:
     twitter: multcloud
     facebook: multcloud
     img: multcloud.png
-
+
```
Comment (phallobst) [inline _data/backup.yml:139]:
```suggestion

```
Just superfluous whitespace...

File: _data/backup.yml [hunk 2]
```diff
@@ -136,7 +136,14 @@ websites:
     twitter: multcloud
     facebook: multcloud
     img: multcloud.png
-
+    
+  - name: Nextcloud
+    url: https://www.nextcloud.com
+    img: nextcloud.png
+    tfa:
+      - totp
```
Comment (phallobst) [inline _data/backup.yml:144]:
```suggestion
      - totp
      - u2f
```
According to the docs, ` u2f` keys are supported.

File: _data/backup.yml [hunk 3]
```diff
@@ -136,7 +136,14 @@ websites:
...
+    doc: https://docs.nextcloud.com/server/17/user_manual/user_2fa.html
+
```
Comment (phallobst) [inline _data/backup.yml:146]:
```suggestion

```
Whitespace again

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
This image comes straight from their favicon, and looks better in my opinion.
nextcloud

**Comment by phallobst** [comment]
Is this exclusively a self-hosted software package? If that's the case, we cannot include it here unfortunately, based on our [exclusion page](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md#self-hosted-services).

If it's not, maybe we should move it into another category. Being a groupware solution, it fits better in "Communication", I guess.

**Comment by RichJeanes** [comment]
They have " hosting partners" that you can sign up with, but it is primarily a self-hosted platform.

**Comment by papaita** [comment]
End

**Pull request closed without merging**